### ISSUE TITLE
Checklist to move PolicyReport API as a new project in kubernetes-sigs org

### ISSUE THREAD

**Issue by anusha94** [kubernetes-sigs/wg-policy-prototypes#120]
As per the discussion during the bi-weekly Office Hours on [March 1st, 2023](https://docs.google.com/document/d/1ihFfEfgViKlUMbY2NKxaJzBkgHh-Phk5hqKTzK-NEEs/edit#heading=h.iaf98bbc3vkq), the team wants to evaluate if moving the API as a `kubernetes-sigs` project is more feasible than writing a KEP to make it as an official K8s API.

**What needs to be done?**

To create a net new project in k-sigs - follow these [Guidelines](https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md#rules-for-new-repositories)
  - create a [new issue](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-repo&template=repo-create.yml&title=REQUEST%3A+%3CCreate+or+Migrate%3E+%3Cgithub+repo%3E) for adding a new project with sig-auth as the sponsoring SIG
  - can work on the [pending issues](https://github.com/kubernetes-sigs/wg-policy-prototypes/issues) in this repo in parallel - not a prerequisite for migration

**Comment by anusha94** [comment]
**Update:**

Per the discussion in the `sig-auth` Office Hours on [June 21, 2023](https://docs.google.com/document/d/1woLGRoONE3EBVx-wTb4pvp4CI7tmLZ6lS26VTbosLKM/view#heading=h.ii52a0istwiv), we will go ahead with a KEP so that it becomes easier for folks to collaborate. Once the KEP is approved, we will start the process to create a new project in `k-sigs`
